commit.go

```diff
@@ -321,7 +322,8 @@ func Push(image string, dest types.ImageReference, options PushOptions) error {
 		return errors.Wrapf(err, "error recomputing layer digests and building metadata")
 	}
 	// Copy everything.
-	err = cp.Image(policyContext, dest, src, getCopyOptions(options.ReportWriter))
+	// TODO: add credsContext
```

Comment:
Yes to this one too.  I need to add params to the pertinent commands to get the creds/certs/tls values and then I'll turn this on.